x-pack/plugins/cases/public/components/types.ts

```diff
@@ -6,3 +6,5 @@
  */
 
 export type { CaseActionConnector } from '../../common/ui/types';
+
+export type FetchAlertDataFunction = (alertIds: string[]) => [boolean, Record<string, unknown>];
```

Comment:
Should this be in `common/ui` instead?

Reply:
Yeap it makes more sense to me. Also, I think it is better if we renamed it to `UseFetchAlertData` so we can understand that the type is about a React hook.